### PR TITLE
Add remote wallpapers

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt [hunk 1]
```diff
@@ -50,6 +51,16 @@ sealed class Wallpaper {
         data class Focus(override val name: String, override val expirationDate: Date? = null) : Remote() {
             override val remoteParentDirName: String = "focus"
         }
```
Comment (MatthewTighe) [inline app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt:53]: Should we possibly remove this from nightly as well, now that there will be a more official remote wallpaper available?

File: app/src/main/java/org/mozilla/fenix/FeatureFlags.kt [hunk 2]
```diff
@@ -87,7 +85,7 @@ object FeatureFlags {
     /**
      * Enables showing the wallpaper functionality.
      */
-    val showWallpapers = Config.channel.isNightlyOrDebug
+    const val showWallpapers = true
```
Comment (MatthewTighe) [inline app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:88]: Do we need this flag at all if it's true?

**Review by MatthewTighe — APPROVED** [review]
Had a couple small questions, but LGTM otherwise. Feel free to address or :shipit:

File: app/src/main/java/org/mozilla/fenix/FeatureFlags.kt (hunk 2 above)
Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:88]: A guess for the moment, it help us to easily turn it off/on if needed it 😅.  Until we feel the feature it's completely mature.

File: app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt (hunk 1 above)
Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt:53]: Yes, we can remove it :)

File: app/src/main/java/org/mozilla/fenix/FeatureFlags.kt (hunk 2 above)
Comment (jonalmeida) [reply app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:88]: When I add a feature flag, I file a ticket that is linked in the comment of the flag to evaluate when to remove the flag.

For example, this is the issue I used for WebAuthn: https://github.com/mozilla-mobile/fenix/issues/19785

Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:88]: 👍🏽

Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:88]: I filed https://github.com/mozilla-mobile/fenix/issues/24050 for it, thanks @jonalmeida!

**Pull request merged**